apps/src/music/views/ImageAttributions.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+
+import musicI18n from '../locale';
+import {ImageAttributionCopyright} from '../player/MusicLibrary';
+
+import moduleStyles from './ImageAttributions.module.scss';
+
+interface ImageAttributionsProps {
+  attributions: ImageAttributionCopyright[];
+}
+
+const creativeCommonsLicenseLinks: {
+  [version: string]: {name: string; url: string};
+} = {
+  CC2: {
+    name: 'CC BY 2.0',
+    url: 'https://creativecommons.org/licenses/by-sa/4.0/deed.en',
+  },
+  CC3: {
+    name: 'CC BY 3.0',
+    url: 'https://creativecommons.org/licenses/by/3.0/deed.en',
+  },
+  CC4: {
+    name: 'CC BY-SA 4.0',
+    url: 'https://creativecommons.org/licenses/by-sa/4.0/deed.en',
+  },
+};
+
+// A small amount of attribution information that will be passed to the copyright
+// dialog.
+const ImageAttributions: React.FunctionComponent<ImageAttributionsProps> = ({
+  attributions,
+}) => {
+  if (attributions.length === 0) {
+    return null;
+  }
+
+  const getMarkdown = (attribution: ImageAttributionCopyright) => {
+    if (creativeCommonsLicenseLinks[attribution.licenseVersion]) {
+      return musicI18n.imageAttributionBodyCreativeCommons({
+        artist: attribution.artist,
+        srcUrl: attribution.src || '',
+        author: attribution.author,
+        licenseName:
+          creativeCommonsLicenseLinks[attribution.licenseVersion]?.name,
```

Comment:
nit: do you need the optional here since you're only here if there is a mapping for this license version?

Reply:
Good catch; this was left over from a previous iteration.

Reply:
Ah, had to leave it in (and add a fallback) to keep the linter happy.